ldi-orchestrator/ldio-connectors/ldio-ldes-client/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioLdesClient.java

```diff
@@ -71,7 +73,13 @@ private synchronized void run() {
 				processModel(memberSupplier.get().getModel());
 			}
 		} catch (EndOfLdesException e) {
+			log.info("SHUTTING DOWN pipeline {} because end of LDES has been reached", pipelineName);
 			shutdownPipeline();
+		} catch (HttpRequestException e) {
+			updateStatus(PipelineStatusTrigger.HALT, StatusChangeSource.AUTO);
+			clientStatusConsumer.accept(ClientStatus.ERROR);
+			log.error("LDES URL unavailable. Client paused: {}", e.getMessage());
+			run();
```

Comment:
Oh, I was not sure anymore if the user should restart the client or pipeline by them self or that the workbench is responsible for this

Reply:
The use case came, it was easily done..